plugins/woocommerce-blocks/assets/js/data/cart/utils.ts

```diff
@@ -142,10 +142,10 @@ export const setIsCustomerDataDirty = debounce(
 );
 
 /**
- * Controls whether to ignore sync events.
+ * Sets whether it is syncing the stores or not.
```

Comment:
I think we could describe better what is happening here. I am lucky to have some context from pairing with Sam, but if I was coming to this fresh I would have no idea what this is for.

Reply:
Done in https://github.com/woocommerce/woocommerce/pull/55295/commits/cc7647b214661c7c445588a6757d64628578536d.